src/components/game/GameField.tsx

```diff
@@ -19,31 +30,60 @@ export const GameField = observer(() => {
         <br />
         Cash: {player.playerMoney}
         <br />
-        Current bet: {game.bet}
+        Bet on player: {game.playerBet}
+        <br />
+        Bet on tie: {game.tieBet}
+        <br />
+        Bet on banker: {game.bankerBet}
+        <br />
+        Total bet: {game.totalBet}
       </p>
       <p>Game round number: {game.gameRound}</p>
       <div className={styles.players}>
         <div className={styles.playerField}>
           {player.playerName}: {baccarat.playerPoints} <br />
           <div className={styles.cards}>
-            <Card card={baccarat.playerCards[0]} />
-            <Card card={baccarat.playerCards[1]} />
+            <Card
```

Comment:
To avoid it you can create extra component where only "fake" cards will be used.

Reply:
upcoming PR